requirements/requirements.sam.txt

```diff
@@ -1,4 +1,2 @@
 rf-segment-anything==1.0
-torch<=2.0.1
```

Comment:
These aren't required at all anymore?

Reply:
I think this will break `pip install "inference[sam]"` since `torch` won't come down anymore.

Reply:
These get constructed from a concat of requirements eg [here](https://github.com/roboflow/inference/blob/main/.release/pypi/inference.gpu.setup.py#L66).

Reply:
pinning torch low here forced a reinstall of torch (which may or may not force reinstall the cuda local packages, need to double check now that we are using the site-packages cuda). I'll double check this when back